{"status":404,"error":"Not Found","exception":"#\u003cActiveRecord::RecordNotFound: Couldn't find Language with 'uid'=:id\u003e","traces":{"Application Trace":[{"exception_object_id":76080,"id":1,"trace":"app/controllers/api/v1/languages_controller.rb:44:in `set_language'"}],"Framework Trace":[{"exception_object_id":76080,"id":0,"trace":"activerecord (7.1.3.4) lib/active_record/core.rb:253:in `find'"},{"exception_object_id":76080,"id":2,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:403:in `block in make_lambda'"},{"exception_object_id":76080,"id":3,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:183:in `block (2 levels) in halting_and_conditional'"},{"exception_object_id":76080,"id":4,"trace":"actionpack (7.1.3.4) lib/abstract_controller/callbacks.rb:34:in `block (2 levels) in \u003cmodule:Callbacks\u003e'"},{"exception_object_id":76080,"id":5,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:184:in `block in halting_and_conditional'"},{"exception_object_id":76080,"id":6,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:598:in `block in invoke_before'"},{"exception_object_id":76080,"id":7,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:598:in `each'"},{"exception_object_id":76080,"id":8,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:598:in `invoke_before'"},{"exception_object_id":76080,"id":9,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:109:in `run_callbacks'"},{"exception_object_id":76080,"id":10,"trace":"actionpack (7.1.3.4) lib/abstract_controller/callbacks.rb:258:in `process_action'"},{"exception_object_id":76080,"id":11,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/rescue.rb:25:in `process_action'"},{"exception_object_id":76080,"id":12,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/instrumentation.rb:74:in `block in process_action'"},{"exception_object_id":76080,"id":13,"trace":"activesupport (7.1.3.4) lib/active_support/notifications.rb:206:in `block in instrument'"},{"exception_object_id":76080,"id":14,"trace":"activesupport (7.1.3.4) lib/active_support/notifications/instrumenter.rb:58:in `instrument'"},{"exception_object_id":76080,"id":15,"trace":"activesupport (7.1.3.4) lib/active_support/notifications.rb:206:in `instrument'"},{"exception_object_id":76080,"id":16,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/instrumentation.rb:73:in `process_action'"},{"exception_object_id":76080,"id":17,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/params_wrapper.rb:261:in `process_action'"},{"exception_object_id":76080,"id":18,"trace":"activerecord (7.1.3.4) lib/active_record/railties/controller_runtime.rb:32:in `process_action'"},{"exception_object_id":76080,"id":19,"trace":"actionpack (7.1.3.4) lib/abstract_controller/base.rb:160:in `process'"},{"exception_object_id":76080,"id":20,"trace":"actionpack (7.1.3.4) lib/action_controller/metal.rb:227:in `dispatch'"},{"exception_object_id":76080,"id":21,"trace":"actionpack (7.1.3.4) lib/action_controller/metal.rb:309:in `dispatch'"},{"exception_object_id":76080,"id":22,"trace":"actionpack (7.1.3.4) lib/action_dispatch/routing/route_set.rb:49:in `dispatch'"},{"exception_object_id":76080,"id":23,"trace":"actionpack (7.1.3.4) lib/action_dispatch/routing/route_set.rb:32:in `serve'"},{"exception_object_id":76080,"id":24,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:51:in `block in serve'"},{"exception_object_id":76080,"id":25,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:131:in `block in find_routes'"},{"exception_object_id":76080,"id":26,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:124:in `each'"},{"exception_object_id":76080,"id":27,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:124:in `find_routes'"},{"exception_object_id":76080,"id":28,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:32:in `serve'"},{"exception_object_id":76080,"id":29,"trace":"actionpack (7.1.3.4) lib/action_dispatch/routing/route_set.rb:882:in `call'"},{"exception_object_id":76080,"id":30,"trace":"rack (3.1.4) lib/rack/etag.rb:29:in `call'"},{"exception_object_id":76080,"id":31,"trace":"rack (3.1.4) lib/rack/conditional_get.rb:31:in `call'"},{"exception_object_id":76080,"id":32,"trace":"rack (3.1.4) lib/rack/head.rb:15:in `call'"},{"exception_object_id":76080,"id":33,"trace":"activerecord (7.1.3.4) lib/active_record/migration.rb:655:in `call'"},{"exception_object_id":76080,"id":34,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'"},{"exception_object_id":76080,"id":35,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:101:in `run_callbacks'"},{"exception_object_id":76080,"id":36,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/callbacks.rb:28:in `call'"},{"exception_object_id":76080,"id":37,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/executor.rb:14:in `call'"},{"exception_object_id":76080,"id":38,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/actionable_exceptions.rb:16:in `call'"},{"exception_object_id":76080,"id":39,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'"},{"exception_object_id":76080,"id":40,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'"},{"exception_object_id":76080,"id":41,"trace":"railties (7.1.3.4) lib/rails/rack/logger.rb:37:in `call_app'"},{"exception_object_id":76080,"id":42,"trace":"railties (7.1.3.4) lib/rails/rack/logger.rb:24:in `block in call'"},{"exception_object_id":76080,"id":43,"trace":"activesupport (7.1.3.4) lib/active_support/tagged_logging.rb:135:in `block in tagged'"},{"exception_object_id":76080,"id":44,"trace":"activesupport (7.1.3.4) lib/active_support/tagged_logging.rb:39:in `tagged'"},{"exception_object_id":76080,"id":45,"trace":"activesupport (7.1.3.4) lib/active_support/tagged_logging.rb:135:in `tagged'"},{"exception_object_id":76080,"id":46,"trace":"activesupport (7.1.3.4) lib/active_support/broadcast_logger.rb:240:in `method_missing'"},{"exception_object_id":76080,"id":47,"trace":"railties (7.1.3.4) lib/rails/rack/logger.rb:24:in `call'"},{"exception_object_id":76080,"id":48,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/remote_ip.rb:92:in `call'"},{"exception_object_id":76080,"id":49,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/request_id.rb:28:in `call'"},{"exception_object_id":76080,"id":50,"trace":"rack (3.1.4) lib/rack/runtime.rb:24:in `call'"},{"exception_object_id":76080,"id":51,"trace":"activesupport (7.1.3.4) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'"},{"exception_object_id":76080,"id":52,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/server_timing.rb:59:in `block in call'"},{"exception_object_id":76080,"id":53,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/server_timing.rb:24:in `collect_events'"},{"exception_object_id":76080,"id":54,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/server_timing.rb:58:in `call'"},{"exception_object_id":76080,"id":55,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/executor.rb:14:in `call'"},{"exception_object_id":76080,"id":56,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/static.rb:25:in `call'"},{"exception_object_id":76080,"id":57,"trace":"rack (3.1.4) lib/rack/sendfile.rb:114:in `call'"},{"exception_object_id":76080,"id":58,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/host_authorization.rb:141:in `call'"},{"exception_object_id":76080,"id":59,"trace":"railties (7.1.3.4) lib/rails/engine.rb:536:in `call'"},{"exception_object_id":76080,"id":60,"trace":"puma (6.4.2) lib/puma/configuration.rb:272:in `call'"},{"exception_object_id":76080,"id":61,"trace":"puma (6.4.2) lib/puma/request.rb:100:in `block in handle_request'"},{"exception_object_id":76080,"id":62,"trace":"puma (6.4.2) lib/puma/thread_pool.rb:378:in `with_force_shutdown'"},{"exception_object_id":76080,"id":63,"trace":"puma (6.4.2) lib/puma/request.rb:99:in `handle_request'"},{"exception_object_id":76080,"id":64,"trace":"puma (6.4.2) lib/puma/server.rb:464:in `process_client'"},{"exception_object_id":76080,"id":65,"trace":"puma (6.4.2) lib/puma/server.rb:245:in `block in run'"},{"exception_object_id":76080,"id":66,"trace":"puma (6.4.2) lib/puma/thread_pool.rb:155:in `block in spawn_thread'"}],"Full Trace":[{"exception_object_id":76080,"id":0,"trace":"activerecord (7.1.3.4) lib/active_record/core.rb:253:in `find'"},{"exception_object_id":76080,"id":1,"trace":"app/controllers/api/v1/languages_controller.rb:44:in `set_language'"},{"exception_object_id":76080,"id":2,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:403:in `block in make_lambda'"},{"exception_object_id":76080,"id":3,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:183:in `block (2 levels) in halting_and_conditional'"},{"exception_object_id":76080,"id":4,"trace":"actionpack (7.1.3.4) lib/abstract_controller/callbacks.rb:34:in `block (2 levels) in \u003cmodule:Callbacks\u003e'"},{"exception_object_id":76080,"id":5,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:184:in `block in halting_and_conditional'"},{"exception_object_id":76080,"id":6,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:598:in `block in invoke_before'"},{"exception_object_id":76080,"id":7,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:598:in `each'"},{"exception_object_id":76080,"id":8,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:598:in `invoke_before'"},{"exception_object_id":76080,"id":9,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:109:in `run_callbacks'"},{"exception_object_id":76080,"id":10,"trace":"actionpack (7.1.3.4) lib/abstract_controller/callbacks.rb:258:in `process_action'"},{"exception_object_id":76080,"id":11,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/rescue.rb:25:in `process_action'"},{"exception_object_id":76080,"id":12,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/instrumentation.rb:74:in `block in process_action'"},{"exception_object_id":76080,"id":13,"trace":"activesupport (7.1.3.4) lib/active_support/notifications.rb:206:in `block in instrument'"},{"exception_object_id":76080,"id":14,"trace":"activesupport (7.1.3.4) lib/active_support/notifications/instrumenter.rb:58:in `instrument'"},{"exception_object_id":76080,"id":15,"trace":"activesupport (7.1.3.4) lib/active_support/notifications.rb:206:in `instrument'"},{"exception_object_id":76080,"id":16,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/instrumentation.rb:73:in `process_action'"},{"exception_object_id":76080,"id":17,"trace":"actionpack (7.1.3.4) lib/action_controller/metal/params_wrapper.rb:261:in `process_action'"},{"exception_object_id":76080,"id":18,"trace":"activerecord (7.1.3.4) lib/active_record/railties/controller_runtime.rb:32:in `process_action'"},{"exception_object_id":76080,"id":19,"trace":"actionpack (7.1.3.4) lib/abstract_controller/base.rb:160:in `process'"},{"exception_object_id":76080,"id":20,"trace":"actionpack (7.1.3.4) lib/action_controller/metal.rb:227:in `dispatch'"},{"exception_object_id":76080,"id":21,"trace":"actionpack (7.1.3.4) lib/action_controller/metal.rb:309:in `dispatch'"},{"exception_object_id":76080,"id":22,"trace":"actionpack (7.1.3.4) lib/action_dispatch/routing/route_set.rb:49:in `dispatch'"},{"exception_object_id":76080,"id":23,"trace":"actionpack (7.1.3.4) lib/action_dispatch/routing/route_set.rb:32:in `serve'"},{"exception_object_id":76080,"id":24,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:51:in `block in serve'"},{"exception_object_id":76080,"id":25,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:131:in `block in find_routes'"},{"exception_object_id":76080,"id":26,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:124:in `each'"},{"exception_object_id":76080,"id":27,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:124:in `find_routes'"},{"exception_object_id":76080,"id":28,"trace":"actionpack (7.1.3.4) lib/action_dispatch/journey/router.rb:32:in `serve'"},{"exception_object_id":76080,"id":29,"trace":"actionpack (7.1.3.4) lib/action_dispatch/routing/route_set.rb:882:in `call'"},{"exception_object_id":76080,"id":30,"trace":"rack (3.1.4) lib/rack/etag.rb:29:in `call'"},{"exception_object_id":76080,"id":31,"trace":"rack (3.1.4) lib/rack/conditional_get.rb:31:in `call'"},{"exception_object_id":76080,"id":32,"trace":"rack (3.1.4) lib/rack/head.rb:15:in `call'"},{"exception_object_id":76080,"id":33,"trace":"activerecord (7.1.3.4) lib/active_record/migration.rb:655:in `call'"},{"exception_object_id":76080,"id":34,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'"},{"exception_object_id":76080,"id":35,"trace":"activesupport (7.1.3.4) lib/active_support/callbacks.rb:101:in `run_callbacks'"},{"exception_object_id":76080,"id":36,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/callbacks.rb:28:in `call'"},{"exception_object_id":76080,"id":37,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/executor.rb:14:in `call'"},{"exception_object_id":76080,"id":38,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/actionable_exceptions.rb:16:in `call'"},{"exception_object_id":76080,"id":39,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'"},{"exception_object_id":76080,"id":40,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'"},{"exception_object_id":76080,"id":41,"trace":"railties (7.1.3.4) lib/rails/rack/logger.rb:37:in `call_app'"},{"exception_object_id":76080,"id":42,"trace":"railties (7.1.3.4) lib/rails/rack/logger.rb:24:in `block in call'"},{"exception_object_id":76080,"id":43,"trace":"activesupport (7.1.3.4) lib/active_support/tagged_logging.rb:135:in `block in tagged'"},{"exception_object_id":76080,"id":44,"trace":"activesupport (7.1.3.4) lib/active_support/tagged_logging.rb:39:in `tagged'"},{"exception_object_id":76080,"id":45,"trace":"activesupport (7.1.3.4) lib/active_support/tagged_logging.rb:135:in `tagged'"},{"exception_object_id":76080,"id":46,"trace":"activesupport (7.1.3.4) lib/active_support/broadcast_logger.rb:240:in `method_missing'"},{"exception_object_id":76080,"id":47,"trace":"railties (7.1.3.4) lib/rails/rack/logger.rb:24:in `call'"},{"exception_object_id":76080,"id":48,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/remote_ip.rb:92:in `call'"},{"exception_object_id":76080,"id":49,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/request_id.rb:28:in `call'"},{"exception_object_id":76080,"id":50,"trace":"rack (3.1.4) lib/rack/runtime.rb:24:in `call'"},{"exception_object_id":76080,"id":51,"trace":"activesupport (7.1.3.4) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'"},{"exception_object_id":76080,"id":52,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/server_timing.rb:59:in `block in call'"},{"exception_object_id":76080,"id":53,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/server_timing.rb:24:in `collect_events'"},{"exception_object_id":76080,"id":54,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/server_timing.rb:58:in `call'"},{"exception_object_id":76080,"id":55,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/executor.rb:14:in `call'"},{"exception_object_id":76080,"id":56,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/static.rb:25:in `call'"},{"exception_object_id":76080,"id":57,"trace":"rack (3.1.4) lib/rack/sendfile.rb:114:in `call'"},{"exception_object_id":76080,"id":58,"trace":"actionpack (7.1.3.4) lib/action_dispatch/middleware/host_authorization.rb:141:in `call'"},{"exception_object_id":76080,"id":59,"trace":"railties (7.1.3.4) lib/rails/engine.rb:536:in `call'"},{"exception_object_id":76080,"id":60,"trace":"puma (6.4.2) lib/puma/configuration.rb:272:in `call'"},{"exception_object_id":76080,"id":61,"trace":"puma (6.4.2) lib/puma/request.rb:100:in `block in handle_request'"},{"exception_object_id":76080,"id":62,"trace":"puma (6.4.2) lib/puma/thread_pool.rb:378:in `with_force_shutdown'"},{"exception_object_id":76080,"id":63,"trace":"puma (6.4.2) lib/puma/request.rb:99:in `handle_request'"},{"exception_object_id":76080,"id":64,"trace":"puma (6.4.2) lib/puma/server.rb:464:in `process_client'"},{"exception_object_id":76080,"id":65,"trace":"puma (6.4.2) lib/puma/server.rb:245:in `block in run'"},{"exception_object_id":76080,"id":66,"trace":"puma (6.4.2) lib/puma/thread_pool.rb:155:in `block in spawn_thread'"}]}}